### PR TITLE
checkout and order summary

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added RajorPay integration
added Checkout Page
Added Meta tags

### CODE REVIEW

**Review by Mohitkumar33 — APPROVED** [review]
clean code. Good to merge

File: src/frontend/pages/checkout/Checkout.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		getAddress(token, userDispatch);
+	}, [token]);
+	return (
+		<div>
+			<Navbar />
+			<Address />
+			{/* <div>
+				{address?.length > 0 ? (
+					<>
+						<button
```
Comment (KanekarSnehal) [inline src/frontend/pages/checkout/Checkout.jsx]: remove comments

**Review by KanekarSnehal — APPROVED** [review]
No issues found, code looks good to merge

File: src/frontend/pages/checkout/Checkout.jsx (hunk 1 above)
Comment (prathmeshjagtap) [reply src/frontend/pages/checkout/Checkout.jsx]: done

**Pull request merged**